apps/src/music/views/MiniMusicPlayer.tsx

```diff
@@ -135,7 +160,15 @@ const MiniPlayerView: React.FunctionComponent<MiniPlayerViewProps> = ({
               />
             </div>
 
-            <div className={moduleStyles.name}>{project.name}</div>
+            <div className={moduleStyles.body}>
+              <div className={moduleStyles.name}>{project.name}</div>
+              {project.labConfig?.music?.packId && (
+                <div className={moduleStyles.details}>
+                  {getPackDetails(project.labConfig.music.packId)?.name} &bull;{' '}
+                  {getPackDetails(project.labConfig.music.packId)?.artist}
```

Comment:
nit: pull out `getPackDetails(project...)` into a local var? (also maybe `project.labConfig.music.packId` since it's used in many other places)

Reply:
Oh, great point.  Had somehow forgotten we get to create new variables inside these `.map` handlers.